content/docs/deployment/csminstallationwizard/src/static/js/ui-functions.js

```diff
@@ -415,8 +415,8 @@ function displayModules(installationType, driverName, CONSTANTS_PARAM) {
 			$(".storageArrays").show();
 			$(".cluster-prefix").show();
 			$(".port-groups").show();
-			$(".resiliency").hide();
```

Comment:
I think you should not remove this line. Because If the version is less than 1.11.0 then we will still see resiliency